contracts/mechs/nevermined/BalanceTrackerNvmSubscription.sol

```diff
@@ -130,6 +129,26 @@ contract BalanceTrackerNvmSubscription is BalanceTrackerBase {
         return 0;
     }
 
+    /// @dev Process mech payment.
+    /// @param mech Mech address.
+    /// @return mechPayment Mech payment.
+    function _processPayment(address mech) internal virtual override returns (uint256 mechPayment, uint256) {
```

Comment:
As far as the understanding goes, all the fees are already accounted for when the subscription is paid for and mech gets its cleared cut.